core/src/avm2/globals/globals.as

```diff
@@ -1,3 +1,7 @@
 include "flash/geom/Point.as"
 include "flash/geom/Rectangle.as"
 include "flash/text/AntiAliasType.as"
+include "flash/filters/BitmapFilter.as"
```

Comment:
Let's sort this list alphabetically (might also be worth to mention it in a comment).